tests/scattering2d/test_numpy_backend_2d.py

```diff
@@ -0,0 +1,184 @@
+import numpy as np
+import pytest
+
+backends = []
+
+from kymatio.scattering2d.backend.numpy_backend import backend
+backends.append(backend)
+
+
+class TestPad:
+    @pytest.mark.parametrize('backend', backends)
+    def test_Pad(self, backend):
+        pad = backend.Pad((2, 2, 2, 2), (4, 4), pre_pad=False)
+
+        x = np.random.randn(1, 4, 4) + 1J * np.random.randn(1, 4, 4)
+
+        z = pad(x)
+
+        assert z.shape == (1, 8, 8)
+        assert z[0, 2, 2] == x[0, 0, 0]
+        assert z[0, 1, 0] == x[0, 1, 2]
+        assert z[0, 1, 1] == x[0, 1, 1]
+        assert z[0, 1, 2] == x[0, 1, 0]
+        assert z[0, 1, 3] == x[0, 1, 1]
+
+        pad = backend.Pad((2, 2, 2, 2), (4, 4), pre_pad=True)
```

Comment:
What is the intended merge order? I see this here, which might conflict with the PR that removes this arg

Reply:
Lets start merging Test1d, Test2d. `pre_pad` we'll worry about after.